### PR TITLE
[OpenLDAP] fix get latestCSN

### PR DESCRIPTION
**Pull request by masahide**
* fix get latestCSN  (Value could not be acquired correctly.)
* fix unit of time

### CODE REVIEW

File: mackerel-plugin-openldap/lib/openldap.go [hunk 1]
```diff
@@ -139,7 +144,7 @@ func (m OpenLDAPPlugin) FetchMetrics() (map[string]interface{}, error) {
 		if err != nil {
 			return nil, err
 		}
-		stat["replication_delay"] = float64(masterTime.UnixNano() - localTime.UnixNano())
+		stat["replication_delay"] = float64(masterTime.UnixNano()-localTime.UnixNano()) / float64(time.Second)
```
Comment (astj) [inline mackerel-plugin-openldap/lib/openldap.go]: [nit]
`masterTime.Sub(localTime).Seconds()` will produce the same value

**Comment by astj** [comment]
Please fix corresponding test too. https://github.com/mackerelio/mackerel-agent-plugins/blob/master/mackerel-plugin-openldap/lib/openldap_test.go#L95
```
[github.com/mackerelio/mackerel-agent-plugins/mackerel-plugin-openldap]$ go test -v ./...
?   	github.com/mackerelio/mackerel-agent-plugins/mackerel-plugin-openldap	[no test files]
=== RUN   TestTransformKeyName
--- PASS: TestTransformKeyName (0.00s)
=== RUN   TestGetStats
--- PASS: TestGetStats (0.00s)
=== RUN   TestLatestCSN
--- FAIL: TestLatestCSN (0.00s)
	openldap_test.go:97: latestCSN = "2017-07-13 09:47:01.963361 +0000 UTC", want "2016-12-05 03:35:38.343893 +0000 UTC"
FAIL
FAIL	github.com/mackerelio/mackerel-agent-plugins/mackerel-plugin-openldap/lib	0.019s
```

File: mackerel-plugin-openldap/lib/openldap.go [hunk 2]
```diff
@@ -83,23 +83,28 @@ func mergeStat(dst, src map[string]float64) {
 
 func latestCSN(sr *ldap.SearchResult) (time.Time, error) {
 	var res time.Time
-	for _, entry := range sr.Entries {
-		csns := make([]time.Time, len(entry.Attributes))
-		for i, attr := range entry.Attributes {
-			v := entry.GetAttributeValue(attr.Name)
```
Comment (astj) [inline mackerel-plugin-openldap/lib/openldap.go]: [memo]
https://godoc.org/gopkg.in/ldap.v2#Entry.GetAttributeValue

File: mackerel-plugin-openldap/lib/openldap.go [hunk 3]
```diff
@@ -83,23 +83,28 @@ func mergeStat(dst, src map[string]float64) {
...
-		sort.Slice(csns, func(i, j int) bool {
-			return csns[i].UnixNano() > csns[j].UnixNano()
-		})
-		res = csns[0]
-		return res, nil
+		csns[i] = t
 	}
-	return res, errors.New("not found CSN")
+	sort.Slice(csns, func(i, j int) bool {
+		return csns[i].UnixNano() > csns[j].UnixNano()
```
Comment (astj) [inline mackerel-plugin-openldap/lib/openldap.go]: [nit]
Maybe you can use [`Time.After`](https://godoc.org/time#Time.After)

**Comment by astj** [comment]
@masahide Thank you. Reviewed!

File: mackerel-plugin-openldap/lib/openldap.go (hunk 2 above)
Comment (masahide) [reply mackerel-plugin-openldap/lib/openldap.go]: forgot the test, oops!

**Comment by syou6162** [comment]
LGTM!

**Pull request merged**